src/api/v1/model_monitors.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+from src.middleware.auth import authenticate_user
+from src.schemas.modelMonitor import ModelMonitor, ModelMonitorCreateDto
+from fastapi import APIRouter, Depends, status
+from src import crud
+from sqlalchemy.orm import Session
+from src.core.db import get_db
+from src.schemas.user import User
+from src.utils.errors import add_error_responses, errors
+
+
+model_monitors_router = APIRouter()
+
+
+@model_monitors_router.post(
+    "/model-monitors",
+    tags=["Model Monitors"],
+    response_model=ModelMonitor,
+    summary="Create a model monitor",
+    status_code=status.HTTP_201_CREATED,
+    responses=add_error_responses([400, 401, 409]),
+)
+async def create_model_monitor(
+    body: ModelMonitorCreateDto,
+    db: Session = Depends(get_db),
+    authenticated_user: User = Depends(authenticate_user),
+) -> ModelMonitor:
+
+    if body is not None:
+        new_model_monitor = crud.model_monitors.create(db=db, obj_in=body)
+        return new_model_monitor
+    else:
+        return errors.bad_request("Body should not be empty")
+
+
+@model_monitors_router.get(
+    "/models/{model_id}/model-monitors",
+    tags=["Model Monitors"],
```

Comment:
If this starts with `/model/...` it should be under models tags

Reply:
Well I was afraid that these endpoints (along with some previous ones) need a different path. I surely don't want to add them to the models Tag, so would `model-monitors/{model_id}` be a better naming?

Reply:
Or `{model_id}/model-monitors`?

Reply:
Why not just `{model_id}/monitors`?

Reply:
Well that's a solution but we discussed with Megas and another possible solution is: `/model-monitors?model_id=1111111`

Reply:
Then why is it under the /model path?

Reply:
Because it's wrong! Needs fixing!

Reply:
You seem a little bit aggressive

Reply:
Maybe the exclamation marks give a wrong impression! 😋